caddyconfig/httpcaddyfile/builtins.go

```diff
@@ -799,10 +799,56 @@ func parseHandle(h Helper) (caddyhttp.MiddlewareHandler, error) {
 }
 
 func parseHandleErrors(h Helper) ([]ConfigValue, error) {
-	subroute, err := ParseSegmentAsSubroute(h)
+	h.Next()
+	args := h.RemainingArgs()
+	expression := ""
+	if len(args) > 0 {
+		expression = ""
+		codes := []string{}
+		for _, val := range args {
+			if strings.HasSuffix(val, "xx") {
```

Comment:
I think we need a bit more validation here. What if the user gives something like `xxx`? We need to validate that the length is 3 and that the first char is actually a digit.

Elsewhere in the codebase, we have code like this to validate:

```go
					if len(arg) == 3 && strings.HasSuffix(arg, "xx") {
						arg = arg[:1]
					}
					statusNum, err := strconv.Atoi(arg)
					if err != nil {
						return d.Errf("bad status value '%s': %v", arg, err)
					}
```